app/src/main/java/com/sapuseven/untis/activities/MainActivity.kt

```diff
@@ -817,6 +819,12 @@ class MainAppState @OptIn(ExperimentalMaterial3Api::class) constructor(
 		items: List<TimegridItem>
 	): List<TimegridItem> {
 		val newItems = mergeItems(items.mapNotNull { item ->
+			if (displayedElement.value?.type?.equals(TimetableDatabaseInterface.Type.SUBJECT.name) == false
```

Comment:
Why is this check necessary? I don't think there's a way to display subject timetables. Even if there's is, normally you should never be able to see the hidden subject and thus can't view its timetable.

Reply:
I do not remember why I've added the check 🦧

Reply:
Let's remove it then ^^